timelock-impl/src/main/java/com/palantir/atlasdb/timelock/clock/ClockSkewEvents.java

```diff
@@ -75,7 +75,7 @@ public void clockSkew(String server, long skew, long minRequestInterval, long du
     }
 
     public void clockWentBackwards(String server, long amount) {
-        log.error("The clock for server {} went backwards by {} nanoseconds",
+        log.debug("The clock for server {} went backwards by {} nanoseconds",
```

Comment:
For these 3 messages, I don't think timelock where this is used logs at DEBUG at all. If we still wanted to monitor incidences of clock skew (e.g. as a precursor to our very contentious #1412) it's probably better to keep these at INFO, or change the default log level in timelock to be DEBUG for this logger specifically.

Reply:
+1 to "change the default log level in timelock to be DEBUG for this logger specifically."